### PR TITLE
[GNA] Renamed plugin and moved it to new place

### PR DESCRIPTION
**Pull request by mryzhov**
### Details:
Rename plugin:
GNAPlugin-> ov_intel_gna_plugin
Move component to the new location

ov_repo/inference-engine/src/gna_plugin -> ov_repo/src/plugins/intel_gna
### Tickets:
 - 71046

### CODE REVIEW

File: src/plugins/CMakeLists.txt [hunk 1]
```diff
@@ -9,3 +9,7 @@ endif()
 if(ENABLE_INTEL_GPU)
     add_subdirectory(intel_gpu)
 endif()
+
+if(ENABLE_GNA)
```
Comment (nkogteva) [inline src/plugins/CMakeLists.txt:13]:
```suggestion
if(ENABLE_INTEL_GNA)
```

File: cmake/coverage.cmake [hunk 2]
```diff
@@ -73,9 +73,9 @@ if (ENABLE_INTEL_GPU)
 endif()
 
 if(ENABLE_GNA)
```
Comment (nkogteva) [inline cmake/coverage.cmake:75]:
```suggestion
if(ENABLE_INTEL_GNA)
```

Comment (mryzhov) [reply cmake/coverage.cmake]: fixed

**Review by nkogteva — APPROVED** [review]

File: .ci/azure/linux.yml [hunk 3]
```diff
@@ -88,7 +88,7 @@ jobs:
       rm -rf $(BUILD_SAMPLES_DIR) ; mkdir $(BUILD_SAMPLES_DIR)
       sudo rm -rf $(TMP_DIR) ; sudo mkdir $(TMP_DIR) ; sudo chmod 777 -R $(TMP_DIR)
       sudo mkdir -p $(SHARE_DIR)
-      sudo apt --assume-yes update && sudo apt --assume-yes install nfs-common
+      sudo apt --assume-yes install nfs-common
```
Comment (azhogov) [inline .ci/azure/linux.yml:91]: should be fixed, incorrect merge

Comment (mryzhov) [reply .ci/azure/linux.yml:91]: https://github.com/openvinotoolkit/openvino/pull/9032

**Pull request merged**